classes/class-wc-connect-nux.php

```diff
@@ -1,5 +1,16 @@
 <?php
 
+if ( ! class_exists( 'Jetpack_Install_Status' ) ) {
+	abstract class Jetpack_Install_Status {
```

Comment:
@v18 personally, I'd just add the class constants to `WC_Connect_Nux`, since to me it's just a mechanism to avoid some "magic" strings. Ultimately, I'm not too picky on this, so if you want to keep the abstract class, go for it. 👍